packages/ag-charts-community/src/module/__snapshots__/optionsModule.test.ts.snap

```diff
@@ -5266,9 +5266,7 @@ exports[`ChartOptions #prepareOptions for COLUMN_CHART_WITH_NEGATIVE_VALUES_EXAM
       "type": "bar",
       "visible": true,
       "xKey": "month",
-      "xKeyAxis": "x",
```

Comment:
Any idea why these snapshots updated? (This is the `column-with-negative-values` example).